56 last changes requested

juanjoseortega-aircury opened the pull request.

cristobaldiaz-aircury commented on assets/_scss/default/body-main-styles.scss:
```diff
@@ -186,6 +186,9 @@ $border-color: #a1a1a1;
 .services-item__content {
   display: none;
   font-size: 18px;
+  div{
```
Missing space before curly bracket

cristobaldiaz-aircury commented on tabs/offering.md:
```diff
@@ -12,30 +11,37 @@ lng_pair: id_offering
-                 <h1 id="title2">Team Skills</h1>
+            <div class="services-item" onclick="showContent(0)">
+                <div class="services-item__head">Types of Projects and Clients we work with<span></span>
+</div>
```
The indentation doesn't look right

cristobaldiaz-aircury reviewed (changes requested):
Is this how the favicon should look like?
![image](https://github.com/curacultura/curacultura-website/assets/105167930/e4dd0678-5c28-4f67-9852-eec27add0f87)
I cannot even see it.

I've updated the list of requirements in the description to make it more readable. Let me know if that's ok. From that list the only item I don't understand is this one, I'm not sure what it means:
- Change 2 types of projects and clients

juanjoseortega-aircury commented:
> Is this how the favicon should look like? ![image](https://private-user-images.githubusercontent.com/105167930/296967750-e4dd0678-5c28-4f67-9852-eec27add0f87.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUzOTQ4NjcsIm5iZiI6MTcwNTM5NDU2NywicGF0aCI6Ii8xMDUxNjc5MzAvMjk2OTY3NzUwLWU0ZGQwNjc4LTVjMjgtNGY2Ny05ODUyLWVlYzI3YWRkMGY4Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExNlQwODQyNDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNjkyMmE3ZDk5MWJkMDMzNzZkYmFjMjEyMTNiODkxYzg3MmFhMzIyODUyZjJkNzc2YTZjZTc1YWU2MjEwNTczJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PPyTg3p77wtA8hOBUUIaMYVZu0yR0rOohsR3oJHglY8) I cannot even see it.
> 
> I've updated the list of requirements in the description to make it more readable. Let me know if that's ok. From that list the only item I don't understand is this one, I'm not sure what it means:
> 
> * Change 2 types of projects and clients

1. About the logo is the same as earlier, i only have removed the background.
2. About the item from the list i have updated it

cristobaldiaz-aircury reviewed (changes requested):
This branch has conflicts with main.

cristobaldiaz-aircury commented on assets/_scss/default/body-main-styles.scss:
```diff
@@ -538,4 +542,4 @@ $border-color: #a1a1a1;
   }
-}
\ No newline at end of file
+}
```
Missing empty line as end of file

cristobaldiaz-aircury commented on assets/_scss/default/body-main-styles.scss:
```diff
@@ -389,6 +390,9 @@ $border-color: #a1a1a1;
   font-size: var(--text-size);
   margin-left: 25px;
 }
+#offering-services div {
```
It would be better to refactor and create a separate `scss` with all the changes for the offerings page. There's already a section for them:
https://github.com/curacultura/curacultura-website/blob/c6e36dc352ba6c3909e68c88059eb1d5fb4c90a6/assets/_scss/default/body-main-styles.scss#L93
